_schemaV2.graphql

```diff
@@ -11732,9 +11732,14 @@ type Me implements Node {
     before: String
     first: Int
 
+    # Include shows near the user's location based on the IP address
+    includeShowsNearIpBasedLocation: Boolean = false
+
     # An IP address, will be used to lookup location
     ip: String
```

Comment:
**suggestion:** maybe we can add a comment here to make sure we don't accidentally send this in the future

Reply:
Good idea! I updated the description to be more accurate.